README.md

```diff
@@ -1,7 +1,5 @@
 # Bot de reserva de mesas
 
-En numerosas ocasiones queremos reservar una mesa en un restaurante para comer con nuestra familia o amigos. Sin embargo, en ocasiones es difícil encontrar el número de teléfono del bar, o está desactualizado en la web, o los empleados están ocupados y nadie lo coge. La idea es automatizar el sistema de reservas mediante un bot de telegram.
+En numerosas ocasiones queremos reservar una mesa en un restaurante para comer con nuestra familia o amigos. Sin embargo, en ocasiones es difícil encontrar el número de teléfono del bar, o está desactualizado en la web, o los empleados están ocupados y nadie contesta. La idea es automatizar el sistema de reservas mediante un bot de telegram que se encargue de toda la gestión de mesas.
```

Comment:
Sigue habiendo detalles de implementación, hasta en el nombre del proyecto: va a ser un bot de Telegram. Eso no es la descripción de un problema a resolver, ni una idea que se pueda implementar.
Adicionalmente, es un proyecto del tipo "almacena y busca". No hay mucho valor añadido en eso desde el punto de vista del servidor; es conveniente que tengáis en cuenta un verdadero modelo de un problema, con una verdadera cantidad de procesamiento para llevarlo a cabo.
En este objetivo estoy pasando ese tipo de problemas o ideas, pero realmente van a ser problemáticos en objetivos ulteriores. Mira, por favor, la versión revisada de la descripción del objetivo 0.